src/test-utils/constants.tsx

```diff
@@ -0,0 +1,29 @@
+import { CurrencyAmount, Token } from '@uniswap/sdk-core'
```

Comment:
These are all token-related testing constants. Would you consider naming this file `src/test-utils/tokens.ts`?

Reply:
Definitely `ts`, not `tsx`.